### PR TITLE
Document the default workspace SA permissions

### PR DESCRIPTION
**Pull request by amisevsk**
### What does this PR do?
Add a doc file for default `DevWorkspace` serviceaccount permissions.

### What issues does this PR fix or reference?
N/A, just occurred to me and is pretty easy to do.

### PR Checklist

- [ ] E2E tests pass (when PR is ready, comment `/test v8-devworkspace-operator-e2e, v8-che-happy-path` to trigger)
    - [ ] `v8-devworkspace-operator-e2e`: DevWorkspace e2e test
    - [ ] `v8-che-happy-path`: Happy path for verification integration with Che

### CODE REVIEW

File: doc/workspace-capabilities.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  - patch
+  - update
+  - delete
+  - list
+  - watch
+```
+
+Additional permissions can be bound to the DevWorkspace ServiceAccount as follows:
+
+1. Find the *DevWorkspace ID* for the DevWorksapce in question. This is available on the `.status.devworkspaceId` field in the object, which can be obtained using `jq`:
```
Comment (JPinkney) [inline doc/workspace-capabilities.md:79]:
```suggestion
1. Find the *DevWorkspace ID* for the DevWorkspace in question. This is available on the `.status.devworkspaceId` field in the object, which can be obtained using `jq`:
```

**Review by JPinkney — APPROVED** [review]
LGTM

File: doc/workspace-capabilities.md (hunk 1 above)
Comment (amisevsk) [reply doc/workspace-capabilities.md:79]: I make this typo 80% of the time. Thanks.

**Review by dkwon17 — APPROVED** [review]

**Comment by amisevsk** [comment]
Needs to be updated after https://github.com/devfile/devworkspace-operator/pull/675

**Pull request merged**